egs/malach/s5/local/chain/tuning/run_tdnn_1i.sh

```diff
@@ -0,0 +1,246 @@
+#!/bin/bash
+
+# Copyright 2019  IBM Corp. (Author: Michael Picheny) Adapted AMI recipe to MALACH corpus
+
```

Comment:
Please put results here, preferably as formatted by local/chain/compare_wer.sh (you may have to change
the script slightly), and rename to 1a.  Also please put the output of steps/info/chain_dir_info.pl, for reference.

Reply:
Problem fixed, I had to use git rm. 

"Here" means where? In this comment? (i.e, I am confused) 

Also all my scoring is sclite based (not kaldi based) does that matter (I would rather not have to rescore everything).

Reply:
normally the results would be near the top of the example scripts, like run_tdnn_1i.sh.  sclite based scoring is fine; you may have to modify local/chain/compare_wer.sh to support that, e.g. grep from the `*.sys` instead of `wer_*`

Reply:
I added the WER results to the top of run_tdnn_1a.sh (renamed per your request). I cannot tell where the output of chain_dir_info.pl can be found, though. I looked in my stdout after train.py was executed and did not see anything remotely like such output. All I see is this:

steps/nnet3/chain/train.py --stage -10 --cmd run.pl --mem 2G --feat.online-ivector-dir exp/nnet3_cleaned/ivectors_train_cleaned_sp_hires_comb --feat.cmvn-opts --norm-means=false --norm-vars=false --chain.xent-regularize 0.1 --chain.leaky-hmm-coefficient 0.1 --chain.l2-regularize 0.00005 --chain.apply-deriv-weights false --chain.lm-opts=--num-extra-lm-states=2000 --egs.dir  --egs.opts --frames-overlap-per-eg 0 --egs.chunk-width 150 --trainer.num-chunk-per-minibatch 128 --trainer.frames-per-iter 1500000 --trainer.num-epochs 9 --trainer.optimization.num-jobs-initial 2 --trainer.optimization.num-jobs-final 4 --trainer.optimization.initial-effective-lrate 0.001 --trainer.optimization.final-effective-lrate 0.0001 --trainer.max-param-change 2.0 --cleanup.remove-egs true --feat-dir data/train_cleaned_sp_hires_comb --tree-dir exp/chain_cleaned/tree_bi --lat-dir exp/chain_cleaned/tri3_cleaned_train_cleaned_sp_comb_lats --dir exp/chain_cleaned/tdnn1i_sp_bi
['steps/nnet3/chain/train.py', '--stage', '-10', '--cmd', 'run.pl --mem 2G', '--feat.online-ivector-dir', 'exp/nnet3_cleaned/ivectors_train_cleaned_sp_hires_comb', '--feat.cmvn-opts', '--norm-means=false --norm-vars=false', '--chain.xent-regularize', '0.1', '--chain.leaky-hmm-coefficient', '0.1', '--chain.l2-regularize', '0.00005', '--chain.apply-deriv-weights', 'false', '--chain.lm-opts=--num-extra-lm-states=2000', '--egs.dir', '', '--egs.opts', '--frames-overlap-per-eg 0', '--egs.chunk-width', '150', '--trainer.num-chunk-per-minibatch', '128', '--trainer.frames-per-iter', '1500000', '--trainer.num-epochs', '9', '--trainer.optimization.num-jobs-initial', '2', '--trainer.optimization.num-jobs-final', '4', '--trainer.optimization.initial-effective-lrate', '0.001', '--trainer.optimization.final-effective-lrate', '0.0001', '--trainer.max-param-change', '2.0', '--cleanup.remove-egs', 'true', '--feat-dir', 'data/train_cleaned_sp_hires_comb', '--tree-dir', 'exp/chain_cleaned/tree_bi', '--lat-dir', 'exp/chain_cleaned/tri3_cleaned_train_cleaned_sp_comb_lats', '--dir', 'exp/chain_cleaned/tdnn1i_sp_bi']
-0.0866605 -0.142017
[info]: CLG not stochastic.
0.27522 -0.568107
HCLGa is not stochastic
0.128172 -0.363415
[info]: final HCLG is not stochastic.